### PR TITLE
Pass shipNames to i18n to get translated names

### PR DESCRIPTION
**Pull request by Werseter**
As suggested on [#poi-viewer](https://github.com/poooi/poi/issues/1596) Discord, this PR delegates translations of names to i18n provided by poi-plugin-translator, instead of #22, being a much cleaner and more reliable solution, without downsides exposed in #23.

### CODE REVIEW

**Comment by Werseter** [comment]
Bump? It's been a while, and this still looks like a best solution to the issues shown in #23

File: lib/notifier.es [hunk 1]
```diff
@@ -31,6 +31,7 @@ export class Notifier {
         this._loader.getSubtitles().then((data) => {
             this._subtitles.ships = data;
             this.__ = I18nService.getPluginI18n();
+            this.__r = (x) => window.i18n.resources.fixedT(x, {lng: I18nService.getLocale()});
```
Comment (KochiyaOcean) [inline lib/notifier.es]: If `poi-plugin-translator` is not installed, `window.i18n.resources.fixedT` will not be available and cause crash.

**Comment by Werseter** [comment]
Isn't resources node being created by poi no matter if poi-plugin-translator is present, and only then plugged in? As far as I recall, getfixedt returns the same input string if translation is not found, but my test case was disabling the translator, not uninstalling and restarting without it.I guess I will confirm this and push yet another PR if needed...

**Comment by KochiyaOcean** [comment]
@Werseter `i18n.resource.__` is preserved (for plugins published before i18n refractor of poi) but `fixedT` is not. I've fixed the plugin by using imported `i18next` instead call `window.i18n.resources`

**Pull request merged**